tools/interactive/interactivetool_bam_iobio.xml

```diff
@@ -1,46 +1,43 @@
-<tool id="interactive_tool_bam_iobio" tool_type="interactive" name="BAM (iobio) Visualisation" version="0.1">
+<tool id="interactive_tool_bam_iobio" tool_type="interactive" name="bam.iobio visualisation" version="0.2">
     <requirements>
-        <container type="docker">qiaoy/iobio-bundle.bam-iobio:1.0-ondemand</container>
+        <container type="docker">anderspitman/bam.iobio.io:latest</container>
```

Comment:
Can we have here a fixed version or a tagged container? Also, dockerhub might restrict pulls and bandwidth and might also remove unused old containers. Any change we can get a copy to quay.io?